v1.0/openapi3/schemas/PersonalIdentifierType.yaml

```diff
@@ -14,4 +14,21 @@ enum:
   - VOTERS_ID
   - UNITED_NATIONS
   - OTHER_ID
-description: "Below are the allowed values for the enumeration.\n- PASSPORT - A passport number is used as reference to a Party.\n\n- NATIONAL_REGISTRATION - A national registration number is used as reference to a Party.\n\n- DRIVING_LICENSE - A driving license is used as reference to a Party.\n\n- ALIEN_REGISTRATION - An alien registration number is used as reference to a Party.\n\n- NATIONAL_ID_CARD - A national ID card number is used as reference to a Party.\n\n- EMPLOYER_ID - A tax identification number is used as reference to a Party.\n\n- TAX_ID_NUMBER - A tax identification number is used as reference to a Party.\n\n- SENIOR_CITIZENS_CARD - A senior citizens card number is used as reference to a Party.\n\n- MARRIAGE_CERTIFICATE - A marriage certificate number is used as reference to a Party.\n\n- HEALTH_CARD - A health card number is used as reference to a Party.\n\n- VOTERS_ID - A voter’s identification number is used as reference to a Party.\n\n- UNITED_NATIONS - An UN (United Nations) number is used as reference to a Party.\n\n- OTHER_ID - Any other type of identification type number is used as reference to a Party."
+description: |
+  Below are the allowed values for the enumeration.\n
+  - PASSPORT - A passport number is used as reference to a Party.\n\n
+  - NATIONAL_REGISTRATION - A national registration number is used as
+  reference to a Party.\n\n
+  - DRIVING_LICENSE - A driving license is used as reference to a Party.\n\n
+  - ALIEN_REGISTRATION - An alien registration number is used as reference to a
+  Party.\n\n
```

Comment:
I think this could be written in the previous line since the desc for some other enum types below are still longer?

Reply:
Sure.